README.md

```diff
@@ -0,0 +1,55 @@
+<div align="center">
+
+# slurm charms
+
+Manage HPC clusters using [Juju](https://juju.is) and the [SLURM](https://slurm.schedmd.com/overview.html) workload manager.
+
+[![CI](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)
+[![Release](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#ubuntu-hpc:matrix.org)
+
+</div>
+
+The `slurm-charms` repository provides many of the SLURM daemons that are required to deploy, configure and manage a SLURM-based HPC cluster:
+
+- [`slurmctld`](./charms/slurmctld/): the central management daemon for SLURM.
+- [`slurmd`](./charms/slurmd): the compute node daemon for SLURM.
+- [`slurmdbd`](./charms/slurmdbd): the database daemon for SLURM.
+- [`slurmrestd`](./charms/slurmrestd/): the REST API interface to SLURM.
+
+## Usage
+
+This should be run using Juju 3.x or greater.
+
+#### Deploy a minimal Charmed SLURM cluster
+
+```shell
+$ juju deploy slurmctld --channel edge
+$ juju deploy slurmd --channel edge
+$ juju deploy slurmdbd --channel edge
+$ juju deploy slurmrestd --channel edge
+$ juju deploy mysql --channel 8.0/edge
+$ juju deploy mysql-router slurmdbd-mysql-router --channel dpe/edge
+
+$ juju integrate slurmctld:slurmd slurmd:slurmctld
+$ juju integrate slurmctld:slurmdbd slurmdbd:slurmctld
+$ juju integrate slurmctld:slurmrestd slurmrestd:slurmctld
+$ juju integrate slurmdbd-mysql-router:backend-database mysql:database
+$ juju integrate slurmdbd:database slurmdbd-mysql-router:database
+```
+
+## Project & Community
+
+`slurm-charms` is a project of the [Ubuntu HPC](https://discourse.ubuntu.com/t/high-performance-computing-team/35988)
+community. It is an open source project that is welcome to community involvement, contributions, suggestions, fixes, and
+constructive feedback. Interested in being involved with the development of the project? Check out these links below:
+
+* [Join our online chat](https://matrix.to/#/#ubuntu-hpc:matrix.org)
+* [Contributing guidelines](./CONTRIBUTING.md)
+* [Code of conduct](https://ubuntu.com/community/ethos/code-of-conduct)
+* [File a bug report](https://github.com/charmed-hpc/slurm-charms/issues)
+* [Juju SDK docs](https://juju.is/docs/sdk)
```

Comment:
How do you feel about using this boilerplate from the doc site landing page for this section?

```suggestion
The Slurm charms are a project of the [Ubuntu High-Performance Computing community](https://ubuntu.com/community/governance/teams/hpc). Interested in contributing bug fixes, patches, documentation, or feedback? Want to join the Ubuntu HPC community? You’ve come to the right place 🤩

Here’s some links to help you get started with joining the community:

* [Ubuntu Code of Conduct](https://ubuntu.com/community/ethos/code-of-conduct)
* [Contributing guidelines](./CONTRIBUTING.md)
* [Join the conversation on Matrix](https://matrix.to/#/#hpc:ubuntu.com)
* [Get the latest news on Discourse](https://discourse.ubuntu.com/c/hpc/151)
* [Ask and answer questions on GitHub](https://github.com/orgs/charmed-hpc/discussions/categories/q-a)
```

Big thing is that I think the nitty gritty development information should be stored within CONTRIBUTING.md. That's where users who want to contribute fixes and features should go to learn more about the tech we work with such as the Juju SDK and charmcraft. The Juju SDK isn't that interesting to someone who just wants to know how to deploy the Slurm charms, or ask a question on GitHub.